Solution1/BusinessLayer/Command/BestWeatherCommand.cs

```diff
@@ -1,15 +1,13 @@
 ﻿using BusinessLayer.Command.Abstract;
 using BusinessLayer.DTOs;
+using BusinessLayer.DTOs.Enum;
```

Comment:
get used to call folder in plural: Dtos, Enums - you save in folders usually not one file

Reply:
+